decoder/expr_function.go

```diff
@@ -297,3 +296,14 @@ func (fe functionExpr) matchingFunctions(prefix string, editRange hcl.Range) []l
 
 	return candidates
 }
+
+func hoverContentForFunction(funcSig schema.FunctionSignature, funcExpr *hclsyntax.FunctionCallExpr) lang.MarkupContent {
```

Comment:
```suggestion
func hoverContentForFunction(name string, funcSig schema.FunctionSignature) lang.MarkupContent {
```

Since we are using only one field (`name)`, we could pass it to the function as a string instead of passing the whole expression.